Final clean up API features (#173)
UniRegensburg/unsere-app-fur-die-universitat-regensburg-bib-buddy: README.md

Bib Buddy offers a platform for concentrated literary work. For that, we create an open atmosphere for your thoughts and ideas during research. Our service makes your literary work as intuitive as never before!

## Functionality
### Your library
Bib Buddy offers the possibility to retrieve data for books by scanning their barcodes, typing their ISBNs or simply adding the data manually in a form. All books can be structured in shelves in your personal library inside the app. Naturally, that library is searchable.

### Your thoughts

> To support with mobile literature work, we offer to save notes relating to a book either in textual form, with your usual formatting options (text style, alignment...) or in auditory form to quickly store a thought in your app.

Reviewer: ```suggestion
To support mobile literature work, the app offers to save notes relating to a book either in textual form, with your usual formatting options (text style, alignment...) or in auditory form to quickly store a thought in your app.
```
Simply sounds better to me.

### Sharing

> Bib Buddy can share your data and notes about a book, a complete shelf or even your whole library as a bibTeX file. This file can be imported from Zotero or other literature software so you can make the best out of the thoughts you gathered within the app!

Reviewer: ```suggestion
With Bib Buddy you can share your data and notes about a book, a complete shelf or even your whole library as a bibTeX file. This file can be imported from Zotero or other literature software so you can make the best out of the thoughts you gathered within the app!
```
I suggest to phrase it like that since you are sharing the data through Bib Buddy - Bib Buddy does not do it by itself, that might be irritating since users might think this is happening permanently in the background. As well it is more precise.

## Team
Bib Buddy is developed by Sabrina Freisleben (@xSabrina), Silvia Ivanova (@silvia-ivanova-github-rep), Sarah Kurek (@SarahKurek), Luis Moßburger (@luismossburger) and Claudia Schönherr (@claudia3).<br><br>
This app was developed during the practical part of the seminar "Advanced Methods Software Engineering" at the Media Informations Group, University of Regensburg by Alexander Bazo.

## Credit
Data about books is provided by an <a href="https://lod.b3kat.de">API of "B3Kat"</a> - a service of German libraries in Bavaria, Berlin and Brandenburg.<br>

> Icons made by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a>

Reviewer: ```suggestion
Icons are made by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a>
```
Sounds better to me.

Author: The text here is the preferred attribution of Flaticon, so I would leave it that way :)
